### PR TITLE
add ubi8.3 image to community stable

### PR DESCRIPTION
**Pull request by anamnavi**
## PR Summary
Add Ubi8.3 image to community stable (previous attempt to add it to preview in this PR ([link](https://github.com/PowerShell/PowerShell-Docker/pull/548)) but that branch has been deleted.

Also image tag syntax doesn't include full tag or date.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

File: release/preview/ubi83/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
```
Comment (RDIL) [inline release/preview/ubi83/docker/Dockerfile:1]: Nit: extra whitespace.

**Review by RDIL — APPROVED** [review]
LGTM

File: release/preview/ubi83/meta.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "IsLinux" : true,
+    "UseLinuxVersion": false,
+    "PackageFormat": "powershell${previewTag}-${PS_VERSION}-1.rh.x86_64.rpm",
+    "SkipGssNtlmSspTests": true,
+    "osVersion": "RedHat Universal Base Image 8.3",
```
Comment (TravisEz13) [inline release/preview/ubi83/meta.json:6]: We can't use RedHat when describing these images.

```suggestion
    "osVersion": "Universal Base Image 8.3",
```

File: release/preview/ubi83/test-deps/meta.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "IsLinux" : true,
+    "UseLinuxVersion": false,
+    "osVersion": "RedHat Universal Base Image 8.3",
```
Comment (TravisEz13) [inline release/preview/ubi83/test-deps/meta.json:4]:
```suggestion
    "osVersion": "Universal Base Image 8.3",
```

File: release/stable/ubi83/meta.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+    "IsLinux" : true,
+    "UseLinuxVersion": false,
+    "PackageFormat": "powershell-${PS_VERSION}-1.rh.x86_64.rpm",
+    "SkipGssNtlmSspTests": true,
+    "osVersion": "RedHat Universal Base Image 8.3",
```
Comment (TravisEz13) [inline release/stable/ubi83/meta.json:6]:
```suggestion
    "osVersion": "Universal Base Image 8.3",
```

File: release/stable/ubi83/test-deps/meta.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+    "IsLinux" : true,
+    "UseLinuxVersion": false,
+    "osVersion": "RedHat Universal Base Image 8.3",
```
Comment (TravisEz13) [inline release/stable/ubi83/test-deps/meta.json:4]:
```suggestion
    "osVersion": "Universal Base Image 8.3",
```

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**